docs/tutorials/cli.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Getting Started with the CLI
+---
+
+import CLITabs from '@theme/CLITabs';
+import TabItem from '@theme/TabItem';
+
+The [Svix CLI](https://github.com/svix/svix-cli) is one of the fastest ways to interact with our API.
+
+With the Svix CLI you can interact with our API, validate webhook payloads, quickly open our docs, and much more.
+
+In this tutorial, we will learn how to install the Svix CLI and use it to send your first webhook message!
+
+## Install the Svix CLI
+
+<CLITabs>
+<TabItem value="homebrew">
+
+To install the Svix CLI with [Homebrew](https://homebrew.sh), run:
+
+
+
+```sh
+brew install svix/svix/svix
+```
+</TabItem>
+<TabItem value="scoop">
+
+To install the Svix CLI with [Scoop](https://scoop.sh/), run:
+
+
+```sh
+scoop bucket add svix https://github.com/svix/scoop-svix.git
+```
+
+```sh
+scoop install svix
+```
+
+</TabItem>
+<TabItem value="macos">
+
+To install the Svix CLI on macOS without Homebrew:
+
+1. Download the latest Darwin tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Darwin_x86_64.tar.gz`
+
+2. Optionally, install the binary in a location where you can execute it globally (e.g., /usr/local/bin).
+
+</TabItem>
+<TabItem value="linux">
+
+To install the Svix CLI on Linux without a package manager:
+
+1. Download the latest Linux tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Linux_x86_64.tar.gz`
+
+3. Run the executable: `./svix`
+
+</TabItem>
+<TabItem value="windows">
+
+To install the Svix CLI on Windows without Scoop:
+
+1. Download the latest Windows tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the `svix_X.X.X_Windows_x86_64.tar.gz` file.
+
+3. Run the `.exe` file
+
+</TabItem>
+</CLITabs>
+
```

Comment:
Maybe add an admonition here to with a note on adding autocomplete?

```suggestion
:::tip Don't miss out on shell completion

After installing the CLI, try running `$ svix completion --help` to learn how to add shell completions to `svix`.
::: 
```

Reply:
I plan on enabling it by default so I'm not sure we need this here as its not directly related to the "story" of using the tool to interact with the api